test/test_punktgruppe.py

```diff
@@ -0,0 +1,76 @@
+import fire
+from fire.api import FireDb
+from fire.api.model import (
+    Punkt,
+    PunktGruppe,
+    EventType,
+)
+
+
+def test_hent_punktgruppe(firedb):
+    """Test at vi kan læse punktgruppe fra databasen"""
+    punktgruppe = firedb.hent_punktgruppe("Aarhus Nivellementstest")
+
+    assert punktgruppe.jessenpunkt.ident == "RDIO"
+    assert punktgruppe.navn == "Aarhus Nivellementstest"
+
+    assert len(punktgruppe.punkter) == 3
+    for p in punktgruppe.punkter:
+        assert p.ident in ("K-63-19113", "K-63-09933", "K-63-09116")
+
+
+def test_hent_fra_punkt(firedb):
+    """Test at en punktgruppe kan findes via et punkt"""
+    punkt = firedb.hent_punkt("RDIO")
+    punktgruppe = punkt.punktgrupper[0]
+    assert isinstance(punktgruppe, PunktGruppe)
+
+
+def test_opret_punktgruppe(firedb, sagsevent, punktfabrik):
+    """Test at en punktgruppe kan oprettes"""
+
+    punkter = [punktfabrik() for _ in range(3)]
+
+    punktgruppe = PunktGruppe(
+        navn=f"Test-{fire.uuid()[0:9]}",
+        punkter=punkter,
+        jessenpunkt=punktfabrik(),
+    )
+
+    sagsevent.eventtype = EventType.PUNKTGRUPPE_MODIFICERET
+    sagsevent.punktgrupper = [punktgruppe]
+    firedb.indset_sagsevent(sagsevent)
+
+
+def test_udvid_punktgruppe(firedb, sagseventfabrik, punktgruppe, punkt):
+    """Test at en punktgruppe kan oprettes"""
+
+    firedb.session.flush()
```

Comment:
Er det korrekt indset, at `firedb.session.flush()` er med her for at commit'e de entiteter, der kommer med som fixtures til test-funktionen, til databasen, inden de kan anvendes som eksisterende objekter i databasen efterfølgende i funktionen?

Reply:
`flush()` er stadiet før `commit()`, men ja, kaldet laves for at stage tingene i databasen. Uden `flush()` bliver id'er og andet administrativt ikke udfyldt. Med `flush()` er data indlæst i databasen (og har overlevet mødet med div. constraints og triggers) men kan stadig rulles tilbage med `rollback()`. Der skal et `commit()` til før data lægges permanent i databasen.